src/test/java/org/opentripplanner/updater/stoptime/TimetableSnapshotSourceTest.java

```diff
@@ -158,10 +136,12 @@ public void testHandleCanceledTrip() throws InvalidProtocolBufferException {
     assertSame(forToday.getTripTimes(tripIndex2), schedule.getTripTimes(tripIndex2));
 
     final TripTimes tripTimes = forToday.getTripTimes(tripIndex);
+
+    /*
     for (int i = 0; i < tripTimes.getNumStops(); i++) {
       assertEquals(PickDrop.CANCELLED, pattern.getBoardType(i));
       assertEquals(PickDrop.CANCELLED, pattern.getAlightType(i));
-    }
+    }*/
```

Comment:
You commented this out because it didn't work? I think the canceled trips are not used in routing currently and therefore I thought they would have the CANCELLED PickDrop for each drop

Reply:
Yes, they are set to SCHEDULED. Not sure what the correct value should be.

Reply:
These lines can be removed. `tripTimes.getRealTimeState()`, which is tested on the next row is used for filtering nowdays.

Reply:
Lines removed.

Reply:
These lines are removed.